### PR TITLE
Colossus Bid Adapter: Add GPP Support and Accept eids from publisher request

### PR DESCRIPTION
**Pull request by bhasker-ddh**
## Type of change

- [x] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

### CODE REVIEW

**Comment by ncolletti** [comment]
@bhasker-ddh changes looks good. the PR has merge conflicts that need to be resolved before we can merge, could you take a look?

**Comment by bhasker-ddh** [comment]
> @bhasker-ddh changes looks good. the PR has merge conflicts that need to be resolved before we can merge, could you take a look?

@ncolletti Thank you for the review. Conflicts are now resolved.

**Review by ncolletti — APPROVED** [review]
looks good!

File: package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "gulp-util": "^3.0.0",
     "is-docker": "^2.2.1",
     "istanbul": "^0.4.5",
-    "karma": "^6.3.2",
+    "karma": "^6.4.3",
```
Comment (ncolletti) [inline package.json:82]: Oh doh, I didn't see this and the package.lock change before approving. @bhasker-ddh could you remove both of these so there are no longer conflicts before merging?

Comment (bhasker-ddh) [reply package.json:82]: Can you check it now @ncolletti

Comment (ncolletti) [reply package.json:82]: @bhasker-ddh The PR still has the `karma` version bump and changes to `package-lock`, could you try removing these again?

**Comment by ncolletti** [comment]
@bhasker-ddh we're close! as a final step, could you merge in the latest master to your branch? That should resolve the latest conflict message we're seeing

**Comment by bhasker-ddh** [comment]
> @bhasker-ddh we're close! as a final step, could you merge in the latest master to your branch? That should resolve the latest conflict message we're seeing

Can you check now @ncolletti

**Pull request merged**